study-builder/tenants/testboston/pages/login.html

```diff
@@ -105,10 +105,12 @@
     var connection = config.connection;
     var prompt = !prompt;
     var showPasswordRecentText = false;
-    var doNotShowRegLink = false;
     var language = config.extraParams.language || 'en';
     var loginUrl = '##BASE_URL##/language-redirect?language=' + language + '&destination=' + encodeURIComponent('login-landing/login');
 
+    // Hide registration link for admin client. Note that param values are passed as strings.
+    var doNotShowRegLink = config.extraParams.is_admin_client === 'true';
```

Comment:
Have you checked that `is_admin_client` gets here as a string? It is set as a boolean on the Angular side.

Reply:
I mean, I saw the comment, but it is weird.

Reply:
Yup, I have checked.

![Screen Shot 2020-12-17 at 2 03 20 PM](https://user-images.githubusercontent.com/5713833/102531658-a5f8e880-4070-11eb-93bf-1840a8714f74.png)